cli/popper/commands/cmd_dot.py

```diff
@@ -0,0 +1,71 @@
+import os
+
+import click
+import hcl
+import popper.utils as pu
+from ..cli import pass_context
+from popper.gha import Workflow
+
+
+
+@click.option(
+    '--wfile',
+    help=(
+        'File containing the definition of the workflow. '
+        '[default: ./github/main.workflow OR ./main.workflow]'
+    ),
+    required=False,
+    default=None
+)
+@click.option(
+    '--recursive',
+    help='Generate .dot file for any.workflow file '
+         'found recursively from current path.',
+    required=False,
+    is_flag=True
+)
+@click.command('dot', short_help='Generates a .dot format for a given workflow'
+               '\n[.dot is a format for graphical representation]')
```

Comment:
the square brackets are used to describe default options. I'd leave this line out.